myapp/db/schema.rb

```diff
@@ -21,13 +22,4 @@
     t.integer "status", default: 0, null: false
     t.index ["title"], name: "index_tasks_on_title"
   end
-
-  create_table "users", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci", force: :cascade do |t|
```

Comment:
so user table is already added at step 13 ... 😅

Reply:
yeah... 😅